tests/test_std.rs

```diff
@@ -100,6 +100,7 @@ fn dedup() {
 
 #[test]
 fn all_equal() {
+    assert!("".chars().all_equal());
```

Comment:
Good one.

Reply:
Yes, it will be a nice addtition. I have added it.

Reply:
Adding a quickcheck test would be a great alternative to either of those (something for another PR)